.github/workflows/main.yml

```diff
@@ -118,6 +116,11 @@ jobs:
           create-args: >-
             python=${{ matrix.python-version }}
           micromamba-version: "1.5.10-0"  # pinned to avoid the breaking changes with mamba and micromamba (2.0.0).
+      - name: Install xclim development version
+        # FIXME: This is a workaround to install xclim v0.53, which is not released.
+        run: |
+          echo "Installing xclim from main branch until version 0.53.0+ is released"
+          python -m pip install git+https://github.com/Ouranosinc/xclim.git@main
```

Comment:
not sure how this should be changed, but commenting so it pops out

Reply:
xclim v0.53 has been released; we can probably remove this entire step.